README-ZH.md

```
@@ -0,0 +1,425 @@
+</h1>
+
+
+<div align="center">
+
+**适用于移动端和边缘设备的快速、轻量级多模态大语言模型推理引擎**
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT)
+[![Documentation](https://img.shields.io/badge/docs-website-blue)](https://ubiquitouslearning.github.io/mllm/)
+[![GitHub Stars](https://img.shields.io/github/stars/UbiquitousLearning/mllm.svg)](https://github.com/UbiquitousLearning/mllm/stargazers)
+
+[📚 文档](https://ubiquitouslearning.github.io/mllm/) • [🚀 快速开始](#quick-starts) • [💡 示例](./examples/) • [🛠️ 安装](#installation)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)